cassandra/README.md

```diff
@@ -29,6 +29,7 @@ INSERT into offices(officeCode, city, phone, addressLine1, addressLine2, state,
 ('1','San Francisco','+1 650 219 4782','100 Market Street','Suite 300','CA','USA','94080','NA');
 ```
 Verify that the data was populated.
+<<<<<<< HEAD
```

Comment:
Looks like you accidentally committed this from a rebase/merge.

Reply:
Deleted. Had an error when merging